### PR TITLE
Extended Support to Enclosure Group for API 1600.

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended Support to Enclosure Group for API 1600. 

### Issues Resolved
N/A

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by chebroluharika — APPROVED** [review]

File: examples/enclosure_group.pp [hunk 1]
```diff
@@ -56,6 +56,7 @@
     }
```
Comment (VenkateshRavula) [inline examples/enclosure_group.pp:56]: As a good practice, update the copy right year in examples and lib files.

Comment (AsisBagga) [reply examples/enclosure_group.pp:56]: @VenkateshRavula Done!

**Review by VenkateshRavula — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 - This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20).
 
 ### Oneview Features supported
+- Encloure Group
```
Comment (chebroluharika) [inline CHANGELOG.md]: Change this to Enclosure Group

**Comment by chebroluharika** [comment]
Please change resource name in CHANGELOG before merging

File: CHANGELOG.md (hunk 2 above)
Comment (AsisBagga) [reply CHANGELOG.md]: done

**Pull request merged**